shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -633,6 +634,16 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
       object.accessibilityCustomActions = accessibilityCustomActions;
     }
 
+    if (node.HasFlag(flutter::SemanticsFlags::kIsLiveRegion)) {
+      NSNumber* identifer = @(node.id);
+      NSString* label = @(node.label.data());
```

Comment:
I wonder if we also need to do this for value?

Reply:
I'm not sure - label/value are fairly distinct on iOS right - do you think we should concatenate them like we do for Android?